### PR TITLE
Main doc update

### PR DESCRIPTION
**Pull request by heitorlessa**
*Issue #, if available:* #9 

*Description of changes:*

* [X] Updates deployment instructions
* [X] Updates instructions on Stripe Integration
* [x] Includes how to delete stack
* [X] Includes notice on free tier
* [x] Includes table of services and explanation
* [x] Export amplify managed resources to SSM Parameter Store
* [x] Standardize SSM Parameter Store parameter names (e.g. /twitch/X) to ease batch gets

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -64,8 +64,8 @@ deploy.booking: ##=> Deploy booking service using SAM
 			--parameter-overrides \
 				BookingTable=$${BOOKING_TABLE_NAME} \
 				FlightTable=$${FLIGHT_TABLE_NAME} \
-				CollectPaymentFunction=/service/payment/collect-function/$${AWS_BRANCH} \
-				RefundPaymentFunction=/service/payment/refund-function/$${AWS_BRANCH} \
+				CollectPaymentFunction=/$${AWS_BRANCH}/service/payment/collectFunction \
```
Comment (sthulb) [inline Makefile]: I’d probably go for a structure of /service/payment/function/collect since it’s narrowing the scope at each /

File: Makefile [hunk 2]
```diff
@@ -64,8 +64,8 @@ deploy.booking: ##=> Deploy booking service using SAM
...
+				RefundPaymentFunction=/$${AWS_BRANCH}/service/payment/refundFunction \
```
Comment (sthulb) [inline Makefile]: Ditto

File: amplify.yml [hunk 3]
```diff
@@ -27,6 +29,17 @@ backend:
...
+        ##
+        # Export key deployed resources to System Manager Parameter Store
+        ##
+        - make export.parameter NAME="/${AWS_BRANCH}/stackName" VALUE=${STACK_NAME}
+        - make export.parameter NAME="/${AWS_BRANCH}/service/amplify/auth/userpool/id" VALUE=${COGNITO_USER_POOL_ID}
+        - make export.parameter NAME="/${AWS_BRANCH}/service/amplify/auth/userpool/arn" VALUE=${COGNITO_USER_POOL_ARN}
+        - make export.parameter NAME="/${AWS_BRANCH}/service/amplify/auth/userpool/clientId" VALUE=${COGNITO_USER_POOL_CLIENT_ID}
+        - make export.parameter NAME="/${AWS_BRANCH}/service/amplify/api/id" VALUE=${GRAPHQL_API_ID}
+        - make export.parameter NAME="/${AWS_BRANCH}/service/amplify/api/url" VALUE=${GRAPHQL_URL}
+        - make export.parameter NAME="/${AWS_BRANCH}/service/amplify/storage/flightTable" VALUE=${FLIGHT_TABLE_NAME}
```
Comment (sthulb) [inline amplify.yml]: Following my first comment, these would be /service/amplify/storage/table/flight

And similar for below

**Comment by heitorlessa** [comment]
Feedback addressed - Merging

**Pull request merged**